src/rules/noUnusedVariableRule.ts

```diff
@@ -95,7 +95,8 @@ class NoUnusedVariablesWalker extends Lint.RuleWalker {
 
     // skip exported variables
     public visitVariableStatement(node: ts.VariableStatement): void {
-        if (this.hasModifier(node.modifiers, ts.SyntaxKind.ExportKeyword)) {
+        if (this.hasModifier(node.modifiers, ts.SyntaxKind.ExportKeyword) ||
+            this.hasModifier(node.modifiers, ts.SyntaxKind.DeclareKeyword)) {
```

Comment:
Will change it!
though it seems it isn't really consistent as for example here it is written the same way I did: https://github.com/palantir/tslint/blob/b08678fe3a2ea28368e6c319336c12a4e1a81064/src/rules/labelPositionRule.ts#L30